### DISCUSSION TITLE
Does PATH need to contain all these entries? (`/mnt/c/system32` etc.)

### DISCUSSION THREAD

**Discussion by einarpersson** [microsoft/WSL#9826] [Q&A] [answered]
Hi!

I notice that my `PATH` includes a bunch of entries that points to my mounted c windows 10 file system. This has the consequence that my zsh autosuggestions suggests a lot of things when doing "/tab"

My question is - are all these entries necessary? Looking at the list I notice VS Code which I of course want to keep, and a few others. But `/Windows`? and `/Windows/system32`?

```
/mnt/c/Windows/system32
/mnt/c/Windows
/mnt/c/Windows/System32/Wbem
/mnt/c/Windows/System32/WindowsPowerShell/v1.0/
/mnt/c/Windows/System32/OpenSSH/
/mnt/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common
/mnt/c/Users/me/scoop/shims
/mnt/c/Users/me/AppData/Local/Microsoft/WindowsApps
/mnt/c/Users/me/AppData/Local/Programs/Microsoft VS Code/bin
```

**Comment by Biswa96** [comment, marked as answer]
> Does PATH need to contain all these entries?

That depends on your use cases.

There are some options

* disable the `/mnt/c/` mount entirely with `automount.enable` option, see https://learn.microsoft.com/en-us/windows/wsl/wsl-config#automount-settings
* disable appending PATH variable with `interop.appendWindowsPath` option, see https://learn.microsoft.com/en-us/windows/wsl/wsl-config#interop-settings. Then change the PATH variable in shell's rc file as needed.

**Reply by einarpersson** [reply]
Looks like `interop.appendWindowsPath` is what I am after.

I guess I can switch that and see what happens..
